src/components/course-calendar/plan.cjsx

```diff
@@ -125,6 +123,7 @@ CoursePlan = React.createClass
 
   componentWillMount: ->
     @subscribeToPublishing(@props.item.plan)
+    @checkRoute()
```

Comment:
_Minor:_ this name is confusing to me; it looked like this line did nothing (I thought `checkRoute()` should return something), not that it has side-effects :smile: